back/spec/tasks/single_use/migrate_deprecated_participation_widgets_spec.rb

```diff
@@ -4,9 +4,7 @@
 
 RSpec.describe 'single_use:migrate_deprecated_participation_widgets' do # rubocop:disable RSpec/DescribeClass
   before_all do
```

Comment:
This one always gets me, but I assume we need `before_all` instead of `before(:all)` like in the rest of the tests here.

Reply:
No, I did that on purpose. I used `before_all` here because we create records in this block. Since `before_all` uses transactions, the records will be removed after the tests run.

Reply:
Thanks - I can never get my head around the difference for some reason :-)